src/customJQuery.js

```diff
@@ -0,0 +1,159 @@
+function $(elem) {
+    const innerObject = {};
+    innerObject.element = document.querySelectorAll(elem);
+    innerObject.array = [];
+
+    innerObject.toString = function() {
+        return innerObject.array.toString();
+    }
+
+    innerObject.addClass = function(...classNames) {
+        
+        innerObject.element.forEach(obj => {
+
+            classNames.forEach( className => {
+                obj.classList.add(className);
+            });
+
+        })
+    };
+
+    innerObject.removeClass = function(...classNames) {
+        
+        innerObject.element.forEach(obj => {
+            classNames.forEach( className => {
+                obj.classList.remove(className);
+            });
+        });
+    };
+
+    innerObject.append = function(...contentData) {
+
+        innerObject.element.forEach(obj => {
+            contentData.forEach(contentInfo => {
+                const tempInnerHtml = obj.innerHTML;
+                obj.innerHTML = tempInnerHtml + contentInfo;
+            });
+        });
+    };
+
+    innerObject.remove = function(selector) {
+
+        innerObject.element.forEach(obj => {
+            if(Object.prototype.toString.call(selector) === '[object Undefined]'){
+                obj.remove();
+            } else {
+                obj.remove(selector);
+            }
+        })
+    };
+
+    innerObject.text = function(str) {
+
+        if (str === undefined){
+            let res = '';
+            innerObject.element.forEach(obj => {
+               res += obj.textContent + '\n';
+            })
+            return res;
```

Comment:
Don't forget to use empty lines to separate blocks of text.

P.S. Just a rule. Always use empty line before return